README.md

```diff
@@ -15,8 +15,17 @@ Before you can run the side, make sure you have [asciidoc](http://www.methods.co
 1. go to <http://127.0.0.1:4000/>
 1. profit!
 
-## deploy
+## Publication Info
 
 Pushing to `master` branch, will trigger a Jenkins CI build, which will deploy to <http://staging.aerogear.org>.
 
 Pushing to `production` branch, deploys the site to <http://aerogear.org>.
+
+### Development Steps
+
+. Make changes in aerogear.org repo on a custom branch
+. Create a pr against master
+. Get the changes reviewed and verified
+. Merge to master (which publishes to staging)
```

Comment:
@laurafitzgerald Our account on Cloudbees deactivate because of lack of activity, so this might not work.
One must manually publish to staging system using our help. No big deal: just press a button

Reply:
I just checked the Jenkins job on Cloudbees. There are no Git hooks, nor periodic polling. Might be a limitation of free Cloudbees account.

Reply:
@aliok thanks for review. I will add some info about cloudbees there. and that we should manually trigger these builds when we want to publish. Is it okay to add a list of people who can be contacted about it?

Reply:
sure. just write you and me.